### PR TITLE
Calculator: Fix copy issue, remove -Infinity

### PR DESCRIPTION
**Pull request by pjhampton**
## Description of new Instant Answer, or changes

_PR update by @moollaza_
- Switch to using `$.keypress` for handling inputs as this normalizes all keys and prevents the need to detect if a modifier key (alt, ctrl, cmd) or non-input key (delete, etc) are pressed
- Add support for entering e, pi, sin, cos, tan, sqrt, log, ln via keyboard 

Fixes: #4120 

## People to notify
@moollaza 

## Testing & Review
To be completed by Community Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [x] Title follows correct format (Specifies Instant Answer + Purpose)
- [x] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Code**
- [ ] Adheres to the [DuckDuckGo Style Guide](https://docs.duckduckhack.com/resources/code-style-guide.html)
- [ ] Behaviour is appropriately tested. If improvement, tests are adequately extended.
- [ ] There is no unnecessary files in place (such as editor config files)
- [ ] There is no API keys / secrets present
- [ ] Tests are passing (run `$ duckpan test <goodie_id>`)
    - Tester should report any failures

**Ready to merge?**

- [ ] Has this IA been deployed to and tested on [beta.duckduckgo.com](https://beta.duckduckgo.com/)?
- [ ] For larger commits, has this been approved be more than one community member?
- [ ] The number of reviews is appropriate for this type of PR
- [ ] The commit message is clear, coherent and fitting

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/calculator

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/calculator/calculator.js [hunk 1]
```diff
@@ -879,18 +879,19 @@ DDH.calculator = DDH.calculator || {};
                      * is looked up in the NOSHIFT_KEYCODES and SHIFT_KEYCODES hashes.
                      */
                     $calcInputTrap.keydown(function(e){
-                        e.preventDefault();
```
Comment (moollaza) [inline share/goodie/calculator/calculator.js]: @pjhampton we still need this, but only when the key pressed is one we want to deal with 👍

Comment (moollaza) [inline share/goodie/calculator/calculator.js]: Actually I think we want to fire it regardless, really if the calc has focus our key listener should be dealing with everything 👍

**Comment by moollaza** [comment]
@pjhampton this wasn't working for me, but I think I've fixed it now. Please give it a try to confirm 👍

**Comment by moollaza** [comment]
@pjhampton actually I found yet another way that doesn't require keeping track of meta, ctrl, alt keys, using keypress :)

**Comment by bsstoner** [comment]
This was released to production 🚀

**Pull request merged**